tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/external/iso_3166_status.txt

```diff
@@ -248,7 +248,7 @@ SG	;	Officially assigned	;	Singapore
 SH	;	Officially assigned	;	Saint Helena, Ascension and Tristan da Cunha
 SI	;	Officially assigned	;	Slovenia
 SJ	;	Officially assigned	;	Svalbard et l'Île Jan Mayen (le)
-SK	;	Officially assigned	;	Slovakia
+SK	;	Formerly used	;	Slovakia
```

Comment:
Why is SK 'formerly used'?

Reply:
Something's wrong here.

https://www.iso.org/obp/ui/#iso:code:3166:SK says **Officially Assigned**

Reply:
@macchiati confusion between language/locale code for SK...

Reply:
reverted